stdlib/grpc/src/main/ballerina/src/grpc/grpc_errors.bal

```diff
@@ -113,17 +113,27 @@ public type ResiliencyErrorType ALL_RETRY_ATTEMPTS_FAILED;
 # Represents all the resiliency-related errors.
 public type ResiliencyError AllRetryAttemptsFailed;
 
+# Represents the reason of calling next after the stream has closed.
+public const STREAM_END_REASON = "{ballerina/grpc}StreamEndError";
+# Represents an error when calling next when the stream has closed.
+public type StreamEndError error<STREAM_END_REASON, Detail>;
```

Comment:
Shall we rename it as `StreamClosedError`

Reply:
Renamed it